functions/helpers.bash

```diff
@@ -13,7 +13,11 @@ COL_MAGENTA=$ESC"35;01m"
 COL_CYAN=$ESC"36;01m"
 COL_LGRAY=$ESC"37;01m"
 COL_DGRAY=$ESC"90;01m"
-export COL_DEF COL_RED COL_GREEN COL_YELLOW COL_BLUE COL_MAGENTA COL_CYAN COL_LGRAY COL_DGRAY
+COL_RESET=$ESC"0m"
```

Comment:
I think COL_DEF is sufficient, no need for COL_RESET

Reply:
Yeah, I was just experimenting. Unrelated question, but what exactly is the 'cond' in `cond_redirect` and `cond_echo` stand for? Conditional?